### PR TITLE
Add final touches

### PR DESCRIPTION
**Pull request by Baayeh**
- Added some styling

### CODE REVIEW

File: src/components/AddBook.jsx [hunk 1]
```diff
@@ -1,66 +1,111 @@
...
           />
           <input
             type="text"
-            className="p-2 border focus:outline-none w-[12.5rem] md:w-[27rem] lg:w-[35rem]"
+            className="p-3 md:p-2 border focus:outline-none"
             placeholder="Author"
             value={author}
             onChange={(e) => setAuthor(e.target.value)}
             required
           />
```
Comment (Whoistolu) [inline src/components/AddBook.jsx]: - [ ] [REQUIRED] For this field where a book gets added, you need to follow the exact styles as used in the zeplin design. Kindly change the font, color, font weight, and every other property the (Add new book, book title, category, add book button) has.
### Your Design
Screenshot 2022-11-08 at 18 45 56

### Zeplin Design
Screenshot 2022-11-08 at 18 59 48

If you click on the `ADD NEW BOOK` for example, you will get a property of it on the right side of your screen. Kindly apply these properties for every content of your design.
### For example
Screenshot 2022-11-08 at 19 01 45

File: src/components/Book.jsx [hunk 2]
```diff
@@ -1,29 +1,91 @@
...
+            onClick={deleteBook}
+          >
+            Remove
+          </button>
+          <button
+            type="button"
+            className="px-2 text-blue-700 text-sm hover:text-blue-900"
+          >
+            Edit
+          </button>
```
Comment (Whoistolu) [inline src/components/Book.jsx]: - [x] REQUIRED] The button (comments, remove, edit) also needs to follow the exact styles as used in the zeplin design. Kindly change the font, color, font weight, and every property they have on zeplin.
- [x] [REQUIRED] Kindly apply the same to the other h-tags (Fiction, dfsdf, sdfsdfsd, chapter 3, current chapter), update-progress-button.

### Your design
Screenshot 2022-11-08 at 18 45 20

### Zeplin Design
Screenshot 2022-11-08 at 19 14 21

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @Baayeh 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @Baayeh,

✔️ Nice styling

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**